Cargo.toml

```diff
@@ -52,7 +52,7 @@ bindgen = { version = "0.57.0", optional = true }
 prost-build = { version = "0.7.0", optional = true }
 
 [package.metadata.docs.rs]
-features = ["pkcs11-provider", "tpm-provider", "tss-esapi/docs", "mbed-crypto-provider", "cryptoauthlib-provider"]
+features = ["pkcs11-provider", "tpm-provider", "mbed-crypto-provider", "cryptoauthlib-provider", "direct-authenticator"]
```

Comment:
No `unix-peer-credentials-authenticator`?

Reply:
I think it will get added anyway because it is in the `default-features`

Reply:
and that there is a `no-default-features` [option](https://docs.rs/about/metadata)